src/js/BoardUI.js

```diff
@@ -0,0 +1,17 @@
+import ColumnUI from './ColumnUI';
+import CardAPI from './CardAPI';
+
+class BoardUI {
+  async render() {
+    this.statuses = await CardAPI.getCardStatuses();
+    this.cards = await CardAPI.getCards();
+    this.statuses.map((status) => {
```

Comment:
maybe it's better to use `forEach` here? we do not assign ColumnUI instances anywhere

Reply:
Fixed in 8563a11

Reply:
i see, everything else is ok
good job